tests/test_rfc64_tin.py

```diff
@@ -0,0 +1,27 @@
+"""Tests of features related to GDAL RFC 64
+
+See https://trac.osgeo.org/gdal/wiki/rfc64_triangle_polyhedralsurface_tin.
+"""
+
+import fiona
+
+from .conftest import  requires_gdal22
+
+
+def test_tin_shp(path_test_tin_shp):
+    """Convert TIN to MultiPolygon"""
+    with fiona.open(path_test_tin_shp) as col:
+        assert col.schema['geometry'] == 'Unknown'
+        features = list(col)
+        assert len(features) == 1
+        assert features[0]['geometry']['type'] == 'MultiPolygon'
```

Comment:
Should we also check the coordinates to see if the conversion has been done as expected?

Reply:
Tests are extended to check for coordinates